api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortBuilderServiceImpl.java

```diff
@@ -64,7 +85,7 @@ public static class SearchTerm {
       this.modifiedTerm = modifiedTerm == null ? "" : modifiedTerm;
       this.endsWithTerms =
           endsWithTerms == null
-              ? new ArrayList()
+              ? Collections.emptyList()
```

Comment:
this would also fix it: `new Arraylist<>()`

Reply:
yes, since the list is immutable.

Reply:
any preference?

Reply:
no preference. thank you